### PR TITLE
Release April 10

### PR DESCRIPTION
**Pull request by matentzn**
New release with new XAO terms, updated anatomy pipeline and your new terms.

### CODE REVIEW

File: patterns/definitions.owl [hunk 1]
```diff
@@ -6067,11 +6284,11 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/XPO_0100182> "abnormal hindlimb bud"^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/XPO_0100182> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000001> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002314> <http://purl.obolibrary.org/obo/XAO_0003066>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
 
-# Class: <http://purl.obolibrary.org/obo/XPO_0100183> (abnormal archenteron roof)
```
Comment (matentzn) [inline patterns/definitions.owl]: @seger Reviews this!

File: patterns/definitions.owl [hunk 2]
```diff
@@ -6067,11 +6284,11 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
...
+# Class: <http://purl.obolibrary.org/obo/XPO_0100183> (abnormal gastrocoel roof plate)
```
Comment (matentzn) [inline patterns/definitions.owl]: Review the clas 100183 above. Does the name change make sense? @seger

File: patterns/definitions.owl [hunk 3]
```diff
@@ -23175,11 +23385,11 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/XPO_0102816> "abnormal morphology of suprachiasmatic nucleus "^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/XPO_0102816> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000051> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/XAO_0004072>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
 
-# Class: <http://purl.obolibrary.org/obo/XPO_0102817> (abnormal morphology of auditory nerve )
+# Class: <http://purl.obolibrary.org/obo/XPO_0102817> (abnormal morphology of vestibulocochlear nerve )
```
Comment (matentzn) [inline patterns/definitions.owl]: Does this make sense? @ seger

File: patterns/definitions.owl [hunk 4]
```diff
@@ -27996,10 +28206,10 @@ AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.ob
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/XPO_0103556> "abnormal anatomical structure morphogenesis in optic vesicle"^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/XPO_0103556> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0001236> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/GO_0009653> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/XAO_0000228>))) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
 
-# Class: <http://purl.obolibrary.org/obo/XPO_0103557> (abnormal anatomical structure morphogenesis in neural fold)
+# Class: <http://purl.obolibrary.org/obo/XPO_0103557> (abnormal anatomical structure morphogenesis in neural fold (obsolete))
```
Comment (matentzn) [inline patterns/definitions.owl]: Does this make sense? @seger

File: patterns/definitions.owl [hunk 5]
```diff
@@ -29633,11 +29843,11 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/XPO_0103814> "decreased length of axon in retinal ganglion cell"^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/XPO_0103814> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000574> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/XAO_0005015> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000050> <http://purl.obolibrary.org/obo/XAO_0004100>))) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
 
-# Class: <http://purl.obolibrary.org/obo/XPO_0103815> (decreased length of cilium in archenteron roof)
+# Class: <http://purl.obolibrary.org/obo/XPO_0103815> (decreased length of cilium in gastrocoel roof plate)
```
Comment (matentzn) [inline patterns/definitions.owl]: Does this make sense @seger

**Comment by matentzn** [comment]
@seger Do not merge this, please very carefully review the file called `patterns/definitions.owl` in the [diff](https://github.com/obophenotype/xenopus-phenotype-ontology/pull/19/files).

File: patterns/definitions.owl (hunk 2 above)
Comment (seger) [reply patterns/definitions.owl]: yes

File: patterns/definitions.owl (hunk 3 above)
Comment (seger) [reply patterns/definitions.owl]: yes

File: patterns/definitions.owl (hunk 5 above)
Comment (seger) [reply patterns/definitions.owl]: yes

File: patterns/definitions.owl (hunk 4 above)
Comment (seger) [reply patterns/definitions.owl]: No. Let me look into it..

File: patterns/definitions.owl [hunk 6]
```diff
@@ -7516,11 +7733,11 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/XPO_0100409> "abnormal archenteron"^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/XPO_0100409> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000001> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002314> <http://purl.obolibrary.org/obo/XAO_0000092>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
 
-# Class: <http://purl.obolibrary.org/obo/XPO_0100410> (abnormal foregut primordium)
+# Class: <http://purl.obolibrary.org/obo/XPO_0100410> (abnormal foregut endoderm)
```
Comment (matentzn) [inline patterns/definitions.owl]: Makes sense @seger

File: patterns/definitions.owl [hunk 7]
```diff
@@ -12759,11 +12976,11 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/XPO_0101184> "abnormal suprachiasmatic nucleus"^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/XPO_0101184> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000001> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002314> <http://purl.obolibrary.org/obo/XAO_0004072>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
 
-# Class: <http://purl.obolibrary.org/obo/XPO_0101185> (abnormal auditory nerve)
+# Class: <http://purl.obolibrary.org/obo/XPO_0101185> (abnormal vestibulocochlear nerve)
```
Comment (matentzn) [inline patterns/definitions.owl]: @seger Sense?

File: patterns/definitions.owl [hunk 8]
```diff
@@ -16483,11 +16693,11 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/XPO_0101814> "abnormal morphology of hindlimb bud "^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/XPO_0101814> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000051> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/XAO_0003066>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
 
-# Class: <http://purl.obolibrary.org/obo/XPO_0101815> (abnormal morphology of archenteron roof )
+# Class: <http://purl.obolibrary.org/obo/XPO_0101815> (abnormal morphology of gastrocoel roof plate )
```
Comment (matentzn) [inline patterns/definitions.owl]: @seger

File: patterns/definitions.owl [hunk 9]
```diff
@@ -17932,11 +18142,11 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/XPO_0102041> "abnormal morphology of archenteron "^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/XPO_0102041> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000051> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/XAO_0000092>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
 
-# Class: <http://purl.obolibrary.org/obo/XPO_0102042> (abnormal morphology of foregut primordium )
+# Class: <http://purl.obolibrary.org/obo/XPO_0102042> (abnormal morphology of foregut endoderm )
```
Comment (matentzn) [inline patterns/definitions.owl]: @seger

File: patterns/definitions.owl (hunk 6 above)
Comment (seger) [reply patterns/definitions.owl]: yes @matentzn

File: patterns/definitions.owl (hunk 7 above)
Comment (seger) [reply patterns/definitions.owl]: yes @matentzn

File: patterns/definitions.owl (hunk 8 above)
Comment (seger) [reply patterns/definitions.owl]: It's ok @matentzn

File: patterns/definitions.owl (hunk 9 above)
Comment (seger) [reply patterns/definitions.owl]: It's ok @matentzn

File: patterns/definitions.owl (hunk 4 above)
Comment (matentzn) [reply patterns/definitions.owl]: Did you find out what happened?

**Comment by matentzn** [comment]
We should also check whether the custom label thing worked..

File: patterns/definitions.owl (hunk 4 above)
Comment (seger) [reply patterns/definitions.owl]: I used the wrong ID for neural fold in src/patterns/data/manual/abnormalBiologicalProcessInLocation.tsv at line 88. It should be XAO:0004087. @matentzn

**Comment by matentzn** [comment]
@seger Pretty sure this can be merged. For the curved entities, label change worked as well.

**Pull request merged**